### PR TITLE
Added configuration parameters for blind merkle tree.

### PR DESCRIPTION
**Pull request by AMarcedone**
This builds on AMarcedone/y2k-598, so please review the differences with that branch only (i.e. the blindtree package).

### CODE REVIEW

**Comment by heronhaye** [comment]
@AMarcedone can you change the base branch to 598?

**Comment by AMarcedone** [comment]
@heronhaye done

File: go/blindtree/values.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+		panic("Unrecognized Value Type")
+	}
+}
+
+type SigID []byte
+type Teamv1HiddenTail struct {
+	_struct struct{} `codec:",toarray"` //nolint
+	SigID   SigID
+	LinkID  sig3.LinkID
+	Seqno   keybase1.Seqno
```
Comment (heronhaye) [inline go/blindtree/values.go]: What was the verdict about including SigID here? Do we need it?

Comment (AMarcedone) [reply go/blindtree/values.go]: Removing both SigID and Seqno as we discussed. cc @maxtaco

**Pull request merged**